portfolio.html

```diff
@@ -0,0 +1,138 @@
+            Веб-сайти
+          </button>
+        </li>
+        <li>
+          <button type="button" class="filter-button">
+            Додатки
+          </button>
+        </li>
+        <li>
+          <button type="button" class="filter-button">
+            Дизайн
+          </button>
+        </li>
+        <li>
+          <button type="button" class="filter-button">
+            Маркетинг
+          </button>
+        </li>
+      </ul>
+    </section>
+
+    <section class="projects">
+      <ul class="projects-list">
+        <li class="projects-item">
+          <img class="project-photo" src="./images/technokryak.jpg" width="370" height="294" alt="technokryak">
+          <h3 class="project-name">Технокряк</h3>
+          <p class="project-position">Веб-сайт</p>
+        </li>
+
+        <li class="projects-item">
+          <img class="project-photo" src="./images/poster.jpg" width="370" height="294" alt="poster">
+          <h3 class="project-name">Постер New Orlean vs Golden Star</h3>
+          <p class="project-position">Дизайн</p>
+        </li>
+
+        <li class="projects-item">
+          <img class="project-photo" src="./images/restorant.jpg" width="370" height="294" alt="restorant">
+          <h3 class="project-name">Ресторан Seafood</h3>
+          <p class="project-position">Додаток</p>
+        </li>
+
+        <li class="projects-item">
+          <img class="project-photo" src="./images/prime.jpg" width="370" height="294" alt="prime">
+          <h3 class="project-name">Проєкт Prime</h3>
+          <p class="project-position">Маркетинг</p>
+        </li>
+
+        <li class="projects-item">
+          <img class="project-photo" src="./images/boxes.jpg" width="370" height="294" alt="boxes">
+          <h3 class="project-name">Проєкт Boxes</h3>
+          <p class="project-position">Додаток</p>
+        </li>
+
+        <li class="projects-item">
+          <img class="project-photo" src="./images/inspirations.jpg" width="370" height="294" alt="inspirations">
+          <h3 class="project-name">Inspiration has no Borders</h3>
+          <p class="project-position">Веб-сайт</p>
+        </li>
+
+        <li class="projects-item">
+          <img class="project-photo" src="./images/limited.jpg" width="370" height="294" alt="limited">
+          <h3 class="project-name">Видання Limited Edition</h3>
+          <p class="project-position">Дизайн</p>
+        </li>
+
+        <li class="projects-item">
+          <img class="project-photo" src="./images/lab.jpg" width="270" height="260" alt="lab">
+          <h3 class="project-name">Проєкт LAB</h3>
+          <p class="project-position">Маркетинг</p>
+        </li>
+
+        <li class="projects-item">
+          <img class="project-photo" src="./images/business.jpg" width="370" height="294" alt="business">
+          <h3 class="project-name">Growing Business</h3>
+          <p class="project-position">Додаток</p>
+        </li>
+      </ul>
```

Comment:
section content must be wrapped in the `div.container` element

Reply:
Container is homework 3